src/app/panels/text-panel/text-panel.component.ts

```diff
@@ -14,6 +15,7 @@ import { EvtIconInfo } from '../../ui-components/icon/icon.component';
 })
 export class TextPanelComponent implements OnInit, OnDestroy {
   @Input() hideEditionLevelSelector: boolean;
+  @Input() hidePageSelector: boolean;
```

Comment:
Ok, it makes sense, but I think is not right to hide the page selector in the case of absence of pages, because there is the possibility that there is some original content in front matter (case in which we create a further dummy page), so I would keep it in all cases.